src/sc_crawler/vendors/_upcloud.py

```
@@ -626,20 +632,279 @@ def inventory_ipv4_prices(vendor):
+                # https://upcloud.com/global/postgresql-managed-databases/
+                "custom_extensions": True,
+                # Managed PostgreSQL docs describe encryption at rest.
+                # https://upcloud.com/docs/products/managed-postgresql/encryption/
+                "disk_encryption": True,
+                # Product page advertises automatic updates with zero downtime.
+                # https://upcloud.com/global/postgresql-managed-databases/
+                "auto_upgrade_versions": True,
+                # Plans include daily full backups (`backup_config.interval`).
+                # https://upcloud.com/docs/products/managed-postgresql/backups/
+                "scheduled_backups": bool(backup_cfg.get("interval")),
+                # PITR retention days from backup_config_pg interval × max_count.
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)